### PR TITLE
[release]linux

### PR DESCRIPTION
**Pull request by k2khalsa**

### CODE REVIEW

File: linux_sql/scripts/host_info.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+cpu_architecture=$( echo "$lscpu_out" | egrep "^Architecture:" | awk '{print $2}' | xargs )
+#save cpu model
+cpu_model=$( echo "$lscpu_out" | egrep "^Model:" | awk '{print $2}' | xargs )
+#save cpu mhz
+cpu_mhz=$( echo "$lscpu_out" | egrep "^CPU\ MHz:" | awk '{print $3}' | xargs )
+#save cpu l2_cache in KB
+l2_cache=$( echo "$lscpu_out" | egrep "^L2\ cache:" | awk '{print $3}' | sed 's/K//' | xargs )
+#save total_mem in KB
+total_mem=$( echo "$free_out" | egrep "^Mem:" | awk '{print $2}' | xargs )
+#save timestamp
```
Comment (jarviscanada) [inline linux_sql/scripts/host_info.sh]: these comments are not necessary as the variable names are self-explanatory

File: linux_sql/scripts/host_usage.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#save memory free in MB
+memory_free=$( echo "$free_out" | egrep "^Mem:" | awk '{print $4*0.001}' | xargs )
+#save cpu_idle
+cpu_idle=$( echo "$stat_out" | egrep "all" | awk '{print $13}' | xargs )
+#save cpu_kernal
+cpu_kernel=$( echo "$stat_out" | egrep "all" | awk '{print $6}' | xargs )
+#save disk_io
+disk_io=$( vmstat -d | egrep "sda" | awk '{print $10}' | xargs )
+#save disk_available
+disk_available=$( df -mT / | egrep "sda2" | awk '{print $5}' | xargs )
```
Comment (jarviscanada) [inline linux_sql/scripts/host_usage.sh]: these comments are not necessary as the variable names are self-explanatory

File: linux_sql/scripts/host_info.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            '$cpu_architecture',
+            '$cpu_model',
+            $cpu_mhz,
+            $l2_cache,
+            $total_mem,
+            '$timestamp');"
+#database insert execution
+psql -h $psql_host -p $psql_port -U $psql_user -d $db_name -c "$insert_cmd"
+
+exit 0
```
Comment (jarviscanada) [inline linux_sql/scripts/host_info.sh]: exit $?

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -1,47 +1,49 @@
...
+  if [ "$(docker ps -a -f name=jrvs-psql | wc -l)" != "2" ]; then
+    echo "Error: Container does not exist"
     exit 1
+    fi
 
-else
-  if [ $1 == "start" ]; then
+  if [ $script_execute == "start" ]; then
     docker container start jrvs-psql
     exit $?
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:39]: what happens if container was not created?

**Review by jarviscanada — APPROVED** [review]
good job. 
please make a master PR so we can merge it into master

File: linux_sql/scripts/psql_docker.sh (hunk 4 above)
Comment (k2khalsa) [reply linux_sql/scripts/psql_docker.sh:39]: it is checked by if statement in line 32

**Pull request merged**